playwright-tests/tests/dashboard.treasury-devdao.near.spec.js

```diff
@@ -8,7 +8,18 @@ test.afterEach(async ({ page }, testInfo) => {
 
 test("should go to trustees dashboard", async ({ page, instanceAccount }) => {
   await page.goto(`/${instanceAccount}/widget/app?page=dashboard`);
+
+  await expect(
+    await page.locator("div.h3").filter({ hasText: "Dashboard" })
+  ).toBeVisible();
+
   await expect(
-    await page.getByRole("heading", { name: "Dashboard" })
+    await page.getByText("Treasury Assets: Sputnik DAO")
   ).toBeVisible();
+
+  await expect(await page.locator("iframe.chart")).toBeVisible();
+  await page.mouse.move(600, 500);
```

Comment:
Can you insert an assertion here on what timestamp that should be shown when hovering the mouse?

Also I am wondering why the y-value is changing in the chart when moving the mouse?

Reply:
it's started animation when chart appears - it shows up a line liike in video